pywbem/cim_operations.py

```diff
@@ -870,6 +949,47 @@ def host(self):
         return host
 
     @property
+    def operation_recorder_enabled(self):
+        """
+        :class:`py:bool`: Operation recorder enablement status for connection.
+
+        This is a writeable property; setting this property will change the
+        operation recorder enablement status accordingly for all active
+        operation recorders.
+
+        This property returns`False` if no operation recorder has been added.
+
+        If any recorder in the recorders list has been enabled, it returns
+        `True`. Otherwise it returns `False`.
+
+        Parameters:
+          value (:term:`boolean`)
```

Comment:
Change `boolean` to `py:bool`.

Reply:
Fixed.  Note that I had tried to fix it and it got stuck in text about 3 lines above. That was removed.